test/model.test.js

```diff
@@ -8491,6 +8491,13 @@ describe('Model', function() {
   });
 });
 
+describe('Check if statics functions that is supplied in schema option is availabe (m0_0a)', function() {
```

Comment:
please remove your iniitials in the title of the unit test describe

```suggestion
describe('Check if statics functions that is supplied in schema option is availabe', function() {
```